owl/SOMA-SAY.owl

```diff
@@ -645,10 +585,9 @@ Let xL, xR be objects filling the locatum, relatum roles of this schema. Then on
     
 
 
-    <!-- http://www.ease-crc.org/ont/SOMA.owl#QueryTask -->
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#QueryingTask -->
 
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#QueryTask">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#IllocutionaryTask"/>
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#QueryingTask">
```

Comment:
As before, axioms for QueryingTask got split up between soma-act and soma-say.

Reply:
See my comment above on assertiontask.